src/test/java/com/entrevistador/orquestador/infrastructure/rest/controller/EntrevistaControllerTest.java

```diff
@@ -43,15 +44,17 @@ void shouldLoadCV_WhenPost() throws IOException {
 
         when(this.solicitudEntrevista.generarSolicitudEntrevista(any(), any())).thenReturn(Mono.empty());
 
-        MultipartBodyBuilder multipartRequest = new MultipartBodyBuilder();
-        multipartRequest.part("file", new ByteArrayResource(file.getBytes())).contentType(MediaType.APPLICATION_PDF);
-        multipartRequest.part("formulario", new FormularioDto()).contentType(MediaType.APPLICATION_JSON);
-
         this.webTestClient
                 .post()
-                .uri(URL.append("/cv").toString())
-                .contentType(MediaType.MULTIPART_FORM_DATA)
-                .bodyValue(multipartRequest.build())
+                .uri(URL.append("/solicitudes-entrevistas?username=test").toString())
+                .contentType(MediaType.APPLICATION_JSON)
+                .bodyValue("{\n" +
+                        "    \"empresa\": \"Tesla\",\n" +
+                        "    \"perfil\": \"Software Engineer Java\",\n" +
+                        "    \"seniority\": \"Senior\",\n" +
+                        "    \"pais\": \"Canada\",\n" +
+                        "    \"descripcionVacante\": \"Acerca del empleo\\nAre you a tech professional \"\n" +
+                        "}")
```

Comment:
Sugerencia.
En vez de un JSON como String hardcoded, crear a nivel de test o tal  vez ya exista una Clase con los 5 atributos. Crear la instancia, cargarlos datos y sustituir el método bodyValue por: 
```suggestion
            .body(Mono.just(instancia), Clase.class)
```